roadmap.md

```diff
@@ -1,10 +1,9 @@
 .NET Core Roadmap
 =================
 
-.NET Core is a general purpose, modular, cross-platform and open source implementation of .NET. It includes runtime, framework, compiler and tools components that support a variety of chip and OS targets. These components can be used together or separately.
+.NET Core is a general purpose, modular, cross-platform and open source implementation of .NET. It includes a runtime, framework libraries, compilers and tools that support a variety of chip and OS targets. These components can be used together or separately.
 
 Major .NET Core components:
-
 - [Base Class Libraries](https://github.com/dotnet/corefx)
 - [CoreCLR runtime and RyuJIT compiler](https://github.com/dotnet/coreclr)
 - [CoreRT runtime and .NET Native compiler](https://github.com/dotnet/corert)
```

Comment:
We shouldn't list CoreRT as a .NET Core component. Instead, we should have a section on native compilation, which is where it should be listed. Our message on that is "Important scenario, just not now."